aws/aws-quick-ak.md

```diff
@@ -0,0 +1,222 @@
+## Amazon Web Services (AWS)
```

Comment:
@webdev-ashishk can you please 
- update the name of the quiz
- and file name instead of `aws-quick-ak` to related 

thanks